ee/clickhouse/views/test/test_clickhouse_trends.py

```diff
@@ -332,3 +320,67 @@ def test_insight_trends_compare(self):
         assert sorted([p["id"] for p in prev_people]) == sorted(
             [str(created_people["p1"].uuid), str(created_people["p2"].uuid)]
         )
+
+
+class ClickhouseTestTrendsGroups(ClickhouseTestMixin, LicensedTestMixin, APIBaseTest):
+    maxDiff = None
+    CLASS_DATA_LEVEL_SETUP = False
+
+    def _create_groups(self):
+        GroupTypeMapping.objects.create(team=self.team, group_type="organization", group_type_index=0)
+        GroupTypeMapping.objects.create(team=self.team, group_type="company", group_type_index=1)
+
+        create_group(team_id=self.team.pk, group_type_index=0, group_key="org:5", properties={"industry": "finance"})
+        create_group(team_id=self.team.pk, group_type_index=0, group_key="org:6", properties={"industry": "technology"})
+        create_group(team_id=self.team.pk, group_type_index=0, group_key="org:7", properties={"industry": "finance"})
+        create_group(
+            team_id=self.team.pk, group_type_index=1, group_key="company:10", properties={"industry": "finance"}
+        )
+
+    @snapshot_clickhouse_queries
+    def test_aggregating_by_group(self):
```

Comment:
Do you mean testing the persons alongside the data or the multiple group type index filtering? Part of the improvements we were making last week was to connect the person querying with the data querying since the numbers should always match and that's relevant in testing them together

Reply:
I mean it's doing two different `TrendsRequest`s below and on top of that has person logic following that.

Trying to do too many things in a test often causes it to become very hard to understand. Then when it breaks it gets deleted or "hacked around" rather than being properly handled.

Reply:
removed second part